fuse_core/include/fuse_core/eigen.h

```diff
@@ -79,6 +80,21 @@ std::string to_string(const Eigen::DenseBase<Derived>& m, const int precision =
   return oss.str();
 }
 
+template <typename Derived>
+bool isSymmetric(const Eigen::DenseBase<Derived>& m,
+                 const typename Eigen::DenseBase<Derived>::RealScalar precision =
+                     Eigen::NumTraits<typename Eigen::DenseBase<Derived>::Scalar>::dummy_precision())
+{
+  return m.isApprox(m.transpose(), precision);
+}
+
+template <typename Derived>
+bool isPSD(const Eigen::DenseBase<Derived>& m)
```

Comment:
You're right. I'm going to change it to `isPositiveDefinite`.

Maybe I should also add a version for `isPSD` (maybe renamed to `isPositiveSemiDefinite`). It could also be the same function with a flag :thinking:

Reply:
Adding a flag with a default parameter is an easy change, so you might as well. That said, I'm not sure where in this system a positive-semi-definite matrix is actually acceptable. Maybe there is one somewhere I'm not thinking of.

Reply:
Ok. Leaving as is for now. If the need comes, then we could add it.